### PR TITLE
winkJS blog post

### PR DESCRIPTION
**Pull request by daniellemaxwell**
This adds a blog post about the winkNLP library that's part of the winkJS collection.

### CODE REVIEW

File: content/en/blog/2024-06-10-nlp-with-javascript-using-winkJS.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
+---
+title: "NLP with Javascript Using winkJS"
+slug: "nlp-with-javascript-using-winkJS"
+date: "2024-06-10T16:06:03-04:00"
+draft: false
+image: img/blog/2024-06-10-performing-nlp-in-javascript/machine-learning-robot.webp
+photo_credit: "Image by [fszalai](https://pixabay.com/users/fszalai-32964329/?utm_source=link-attribution&utm_medium=referral&utm_campaign=image&utm_content=7770312) from [Pixabay](https://pixabay.com//?utm_source=link-attribution&utm_medium=referral&utm_campaign=image&utm_content=7770312)"
```
Comment (daniellemaxwell) [inline content/en/blog/2024-06-10-nlp-with-javascript-using-winkJS.md:7]: To be honest, I'm not in love with this image, but gave up on finding something that felt a bit more in line with the topic.

File: content/en/blog/2024-06-10-nlp-with-javascript-using-winkJS.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+<!--more-->
+
+## What is winkNLP?
+In a blog post about [machine learning libraries written in JavaScript](/blog/5-javascript-libraries-to-use-for-machine-learning/), the winkNLP library by winkJS was recommended. It has a multilingual tokenizer, can work with sentence boundary detection (sbd), perform sentiment analysis, and provide part-of-speech tagging out the box. The library has additional tools that can provide a Flesch reading score, measure similarity, flag stop words, and so much more.
+
+## How does it work?
+winkNLP has a readDoc method where text passed to it may be split into tokens, sentences, or entities. It's possible define entities by using the learnCustomEntities method. Once the text is divided up, it can be accessed by the **its** or **as** helpers to make development a breeze.
+
+Want to get the type for each character in a text collection? Use its.type and get an array specifying the type for each token.
```
Comment (pdeziel) [inline content/en/blog/2024-06-10-nlp-with-javascript-using-winkJS.md:24]:
```suggestion
Want to get the type for each character in a text collection? Use `its.type` and get an array specifying the type for each token.
```

File: content/en/blog/2024-06-10-nlp-with-javascript-using-winkJS.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+
+Need to create a frequency table for sentences, tokens, or entities? as.freqTable will return a frequency table in descending order.
```
Comment (pdeziel) [inline content/en/blog/2024-06-10-nlp-with-javascript-using-winkJS.md:26]:
```suggestion
Need to create a frequency table for sentences, tokens, or entities? `as.freqTable` will return a frequency table in descending order.
```

**Review by pdeziel — APPROVED** [review]
Great post! It feels very distilled down to the components that a JS dev would care about which I like. I also really like the animated GIF - it helps to see how things are working visually to me.

**Pull request merged**